org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/CompletionRanking.java

```diff
@@ -0,0 +1,97 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Microsoft Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Microsoft Corporation - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.jdt.ls.core.internal.handlers;
+
+import java.util.Map;
+
+/**
+ * The ranking result of each completion proposal.
+ */
+public class CompletionRanking {
+	/**
+	 * The score of the completion proposal. Score will be added to the proposal's relevance field.
+	 */
+	private int score;
+
+	/**
+	 * A decorator that can be used as a prefix of the completion item label. Providers can use this
+	 * field to highlight the completion items on demand.
+	 */
+	private char decorator;
+
+	/**
+	 * A map data structure that will be appended to completion item's data field. When a completion
+	 * item is selected, the selected item will be passed to the provider. Providers can use the stored
+	 * data to do post process on demand.
+	 */
+	private Map<String, String> data;
+
+	/**
+	 * The score of the completion proposal. Score will be added to the proposal's relevance field.
+	 * <p>
+	 * The acceptable values are [0, 100]. When adding to the relevance, values larger than 100 will
```

Comment:
Do we need to add constants to refer 0 and 100 as MIN_SCORE, MAX_SCORE? Just in case you change the range?

Reply:
Yes, we can add them